draft-ietf-taps-interface.md

```diff
@@ -616,7 +616,52 @@ particular instance. While this does restrict path selection, it is broader than
 requiring specific PvD instances or interface instances, and should be preferred
 over these options.
 
+### Direction of communication
+
+This property specifies whether an application wants to use the connection for sending and/or receiving data.  Possible values are:
+
+Bidirectional (default):
+: The connection must support sending and receiving data
+
+Unidirectional send:
+: The connection must support sending data
+
+Unidirectional receive:
+: The connection must support receiving data
+
+In case a unidirectional connection is requested, but unidirectional connections are not supported by the transport protocol,
+the system should fall back to bidirectional transport.
+
 
+### Bounds on Send or Receive Rate
+
+This property specifies an upper-bound rate that a transfer is not expected to
+exceed (even if flow- and congestion control allow higher rates), and/or a
+lower-bound rate below which the application does not deem
+a data transfer useful. It is given in bits per second.
+
+
+### TCP-specific Property: User Timeout
+
+This property specifies, for the case TCP becomes the chosen transport protocol:
+
+User Timeout:
+: the time after which the connection will be aborted if data could not be delivered
+
+Advertised User Timeout:
+: a time value to be advertised via the User Timeout Option (UTO) for the TCP at the remote endpoint
+to adapt its own user User Timeout value accordingly
+
+User Timeout Enabled:
+: a boolean (default false) to control whether the UTO option is enabled for a
+connection. This applies to both sending and receiving.
+
+Changeable:
+: a boolean (default true) which controls whether the User Timeout may be changed
+based on a UTO option received from the remote peer. This boolean becomes false when
+the 'User Timeout' is used.
+
+All of the above parameters are optional.
```

Comment:
Same question as above: Should this be in Section 9.1? That's where we have all the other timeouts etc.

Reply:
I agree! will do